liblangutil/ErrorReporter.h

```diff
@@ -33,6 +33,13 @@
 namespace solidity::langutil
 {
 
+// Unique identifiers are used to tag and track individual error cases.
```

Comment:
Please use `/**` (doxygen)

Reply:
I used `///`, OK? It saves two lines :).

Reply:
Usually we use `/**` for classes and `///` for functions.

Reply:
Done.